### DISCUSSION TITLE
Built inRP에서 URP로 변경시 캐릭터 렌더 오류 문의

### DISCUSSION THREAD

**Discussion by hjjwon** [naverz/zepeto-studio-kor#678] [Issue Report (Legacy)]
버전 2020.3.9f1
Built inRP를  URP로 변경시키고 기존에 있던 object도 다 URP로 쉐이더 적용시켰을시 다른 오브젝트에는 변경된 렌더가 잘 적용되지만
제페토 캐릭터를 불러오기 하였을시엔 아래 사진 처럼  핑크색으로 보입니다 어떻게 해결 가능할까요?ㅠ
![20220429_104255](https://user-images.githubusercontent.com/102894524/165873203-fb3ffc1e-4278-4842-8dd5-ddcc1cb1e41a.png)
 
프로젝트 세팅은 아래와 같고요
![20220429_104307](https://user-images.githubusercontent.com/102894524/165873210-289349b6-ca6d-42f0-8063-46a957775623.png)

아래와 같은 방법으로 기존 오브제에 렌더 적용시켰습니다 
![20220429_104313](https://user-images.githubusercontent.com/102894524/165873208-1e2546da-502a-4f65-9de8-1cf7128075c6.png)

**Comment by LeeYongSu1** [comment]
아직 URP는 지원이 안되는 걸로 알고 있습니다

**Reply by hjjwon** [reply]
아하 그렇군요 ㅠㅠ 감사합니다!!
